config/routes.rb

```diff
@@ -61,6 +61,18 @@
       resources :media
     end
 
+    resources :settings
+
+    resources :quickbooks do
+      collection do
+        get :authenticate
+        get :oauth_callback
+        get :disconnect
+      end
+    end
+
+    resources :transactions
```

Comment:
Routes should also be namespaced, so /admin/accounting/transactions and /admin/accounting/quickbooks.